guides/tools/soft-phone.md

```diff
@@ -0,0 +1,36 @@
+# Soft-phone
+
+Soft-phone allows you you to make a calls over the Internet using a general purpose computer rather than dedicated hardware. Mostly we use it to make calls to Twilio endpoints. There are lot of different clients, but this doc will explain how to configure Zoiper client.
+
+## Download
+
+To download Zoiper visit https://www.zoiper.com, here you can choose any OS.
+
+## SIP domain
+
+Before app configuration we have to create SIP domain on [Twilio side](https://www.twilio.com/console/voice/sip/endpoints/add). Here you have to fill in following fields
```

Comment:
Before you configure the Zoiper client you have to create SIP domain on [Twilio ](https://www.twilio.com/console/voice/sip/endpoints/add). Fill in following fields:
- FRIENDLY NAME - preferably your name
- SIP URI - URI which you will need to use for Zoiper configuration later
- REQUEST URL - URL to your application which will handle API request (see [how to configure ngrok](https://github.com/DVELP/cookbook/blob/master/guides/ngrok.md) to get public url from your localhost)